src/common/header/shared.h

```diff
@@ -151,7 +151,7 @@ typedef int fixed16_t;
 
 struct cplane_s;
 
-extern vec3_t vec3_origin;
+static vec3_t vec3_origin = {0, 0, 0};
```

Comment:
That might be (I do not mind reverse this specific change).

Reply:
or do we accidentally somehow link in shared.c twice?
(but then there should probably more warnings like that about all the other stuff that's implemented in there?)

Reply:
So I was looking at the project and the common library, dedicated server and the client compiles it.

Reply:
hm yeah the client (quake2 executable, also q2ded) compiles it, and each library (game.so, ref_\*.so) compiles it.
but the libs aren't linked, they're only dlopen()'d, so the compiler shouldn't know about that and it shouldn't leak out of the libs anyway?
what "common library" are you referring to? I only know that common/ folder that is compiled into all these things, but no separate lib

Reply:
Yes in fact sorry for the confusion I mean module (so game.so as you said).

Reply:
anyway, we can leave this change in, it shouldn't hurt and might even make a few things faster as it can be inlined as 3 floats or whatever makes sense to the compiler in the places it's used.
(maybe it should also be `const`?)

but I still think the compiler/analyzer warning (error?) is a false positive - might be worth reporting there :)

Reply:
Yes LLVM sanitizer evolves quite a few :-) maybe it had been fixed in later releases :-)

Reply:
> and might even make a few things faster as it can be inlined as 3 floats
> or whatever makes sense to the compiler in the places it's used.

I remember a case in Sin where we did VectorClear() on some local
vecs (which inlining VectorCopy(vec3_origin,x) would do), and the end
result was bugged. However, doing VectorCopy(vec3_origin,x) with
vec3_origin being opaque was fine. So I suggest that you guys leave
this as is.

Reply:
not sure what exactly you did there, but making it static shouldn't break any code (except if your compiler is broken)